examples/golang/simple/go.sum

```diff
@@ -1,62 +0,0 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
same as above

Reply:
sorry for my misunderstanding. this file is required and should be moved to `examples/golang-http/simple`. It looks like this in local but I don't know why it's shown as deleted in browser. Or there is something wrong with me?

Reply:
i think its a case of using `mv` rather than `git mv`, or perhaps forgetting to used `git add` after moving - either way i think we do want/need the go.sum files

Reply:
seems it was `.gitignore` at fault